app/script/auth/util/RuntimeUtil.js

```diff
@@ -1,19 +1,60 @@
-import '../../util/Environment';
+/*
+ * Wire
+ * Copyright (C) 2017 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import platform from 'platform';
 
 export default class RuntimeUtil {
+  static BROWSER_NAME = {
+    CHROME: 'Chrome',
+    EDGE: 'Microsoft Edge',
+    ELECTRON: 'Electron',
+    FIREFOX: 'Firefox',
+    OPERA: 'Opera',
+    WIRE: 'Wire',
+  };
+
+  static PLATFORM_NAME = {
+    MACINTOSH: 'Mac',
+    WINDOWS: 'Win',
```

Comment:
`Windows`, `macOS`

Reply:
No, the code would fail if I do that, because macOS is not part of the User Agent string. I tried to be compatible to our current code.

Reply:
makes sense then :)